raiden/network/transport/matrix/utils.py

```diff
@@ -294,12 +300,11 @@ def _user_id(self) -> str:
         return user_id
 
     def _fetch_user_presence(self, user_id: str) -> UserPresence:
-        if user_id not in self._userid_to_presence:
-            try:
-                presence = UserPresence(self._client.get_user_presence(user_id))
-            except MatrixRequestError:
-                presence = UserPresence.UNKNOWN
-            self._userid_to_presence[user_id] = presence
+        try:
+            presence = UserPresence(self._client.get_user_presence(user_id))
+        except MatrixRequestError:
+            presence = UserPresence.UNKNOWN
```

Comment:
The PFS filters routes by the node's status being "online" and nothing else. If refreshing the presence fails for some reason, wouldn't it be safe to assume that the node is online? worst case scenario is that the route will be provided but Raiden will not be able to move the transfer forward so we just wait for it to expire (in case we were wrong and the node is actually offline).

If you decide to take my suggestion, i would also document reasoning on the online assumption.

Reply:
The PR goes in a similar direction, but currently we use UNKNOWN as a default all over the transport. I think in this case if we hit a MatrixRequestError we'll retry max 30 seconds later. Changing the default is something I have thought about as well, but I'd rather do that in another PR if that's ok.

Reply:
Makes sense as well.